### ISSUE TITLE
Indent breaks pp and emacs-lisp-mode indentation.

### ISSUE THREAD

**Issue by ruediger** [nicferrier/emacs-noflet#6]
Emacs lisp only supports simple indent statements: (info "(elisp) Indenting Macros").  Setting indent to `((&whole 4 &rest (&whole 1 &lambda &body)) &body)` not only breaks indentation in lisp mode but also breaks the pretty printer.

E.g.,

```
λ> (pp-to-string '(noflet a b c))
*** Eval error ***  Invalid function: ((&whole 4 &rest (&whole 1 &lambda &body)) &body) 
```

Setting it back to `(indent 1)` fixes the issue.

**Comment by gempesaw** [comment]
I was having the same issue; googling and happening to find the right helpfiles led me to the following that makes the indentation declaration valid:

``` elisp
(require 'cl-indent)
(setq lisp-indent-function 'common-lisp-indent-function)
```

**Comment by nicferrier** [comment]
somewhat fixed in 08a0932998408746eb14eb5a65149247510173c2
